worldengine/draw.py

```diff
@@ -477,3 +554,8 @@ def draw_ancientmap_on_file(world, filename, resize_factor=1,
                     draw_biome, draw_rivers, draw_mountains, draw_outer_land_border, 
                     verbose)
     img.complete()
+
+def draw_scatter_plot_on_file(world, filename):
+    img = ImagePixelSetter(512, 512, filename)
```

Comment:
If the plot is purely used for informational purposes, I guess it might be ok. If others are fine with it, then do it the way you like.
(I love the huge maps, however.^^)